src/controllers/check.your.answers.controller.ts

```diff
@@ -32,8 +32,13 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
     logger.debug(`POST ${config.CHECK_YOUR_ANSWERS_PAGE}`);
 
     const transaction: Transaction = await postTransaction(req.session as Session);
+    logger.info(`Transaction created, ID: ${transaction.id}`);
+
     const overseaEntity: OverseasEntityCreated = await createOverseasEntity(req.session as Session, transaction.id as string);
-    await closeTransaction(req.session as Session, transaction.id as string, overseaEntity.id);
+    logger.info(`Overseas Entity Created, ID: ${overseaEntity.id}`);
+
+    const responseClosedTransaction = await closeTransaction(req.session as Session, transaction.id as string, overseaEntity.id);
+    logger.info(`Transaction Closed,  ${JSON.stringify(responseClosedTransaction)}`);
```

Comment:
logger.info(`Transaction Closed, ${JSON.stringify(responseClosedTransaction)}`);

i would just log the transactionId here instead of the whole object
logger.info(`Transaction Closed, id ${responseClosedTransaction.id}`);

Reply:
Thanks Mark, agreed but first we need to change the response coming from the SDK. It will be done in a future story